src/bindings/python/src/pyopenvino/graph/model.cpp

```diff
@@ -50,6 +50,16 @@ static ov::SinkVector cast_to_sink_vector(const std::vector<std::shared_ptr<ov::
     return sinks;
 }
 
+static ov::SinkVector cast_to_sink_vector(const ov::OutputVector& outputs) {
+    ov::SinkVector sinks;
+    for (const auto& output : outputs) {
+        auto sink = std::dynamic_pointer_cast<ov::op::Sink>(output.get_node_shared_ptr());
+        OPENVINO_ASSERT(sink != nullptr, "Output node handle {} is not instance of Sink");
```

Comment:
```suggestion
        OPENVINO_ASSERT(sink != nullptr, "Output {} is not instance of Sink");
```
?

What `{}` means in error message?

Reply:
actually nothing, @almilosz removed it